coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanParser.scala

```diff
@@ -64,7 +65,8 @@ object LogicalPlanParser {
 
   private def periodicSeriesToQuery(periodicSeries: PeriodicSeries): String = {
     // Queries like sum(foo) should not have window even though stale lookback is present
-    s"${rawSeriesLikeToQuery(periodicSeries.rawSeries, false)}"
+    val at = periodicSeries.atMs.fold("")(atMs => s" $At${atMs / 1000}")
```

Comment:
What do you think about using `map` and `orElse` here? `fold` implies `atMs` is a sequence, I think.